src/table/body-cell/styles.scss

```diff
@@ -72,7 +74,8 @@ $cell-offset: calc(#{awsui.$space-m} + #{awsui.$space-xs});
   @for $i from 0 through $max-nesting-levels {
     &.expandable-level-#{$i} {
       > .body-cell-content {
-        margin-inline-start: calc($offset-padding + $i * $cell-offset);
```

Comment:
This is the margin between expand toggle and cell content. Using half padding half margin to give space for the focus outline.